config/real.yaml

```diff
@@ -97,13 +98,15 @@
 /radio:
```

Comment:
since I changed the name of this, maybe it should now be 2 of them? one sim_radio, one network_radio?

Reply:
added a TODO, wasn't confident exactly what you meant by this? plus I feel that this change belongs in a more radio-relevant PR